content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -46,7 +46,7 @@ management policies to determine some placement preferences on the node.
 ### Configuration
 
 The CPU Manager policy is set with the `--cpu-manager-policy` kubelet
-option. There are two supported policies:
+flag or the `cpuManagerPolicy` field in [KubeletConfiguration](/docs/reference/config-api/kubelet-config.v1beta1/). There are two supported policies:
```

Comment:
Please avoid long lines like this by manually wrapping them. This will help the downstream localization efforts.